app/components/InventoryOverview/index.js

```diff
@@ -220,27 +221,26 @@ const InventoryOverview = ({ navigation }) => {
             },
           };
         });
-        let geoJSON = {
+        const geoJSON = {
           type: 'FeatureCollection',
           features: featureList,
         };
-        let fileName = `Tree Mapper GeoJSON ${inventory.inventory_id}.json`;
-        let path = `${
-          Platform.OS == 'ios' ? RNFetchBlob.fs.dirs.DocumentDir : RNFetchBlob.fs.dirs.DownloadDir
-        }/${fileName}`;
-        RNFetchBlob.fs
-          .writeFile(path, JSON.stringify(geoJSON), 'utf88')
-          .then((success) => {
-            alert(
-              `GeoJSON file export in ${Platform.OS == 'ios' ? 'document' : 'download'} directory`,
-            );
+        const options = {
+          url: `data:application/json;name=treemapper-geojson-${inventory.inventory_id}.json;base64,` + toBase64(JSON.stringify(geoJSON)),
+          message: i18next.t('label.inventory_overview_export_json_message'),
+          title: i18next.t('label.inventory_overview_export_json_title'),
+          filename: `Tree Mapper GeoJSON ${inventory.inventory_id}`,
```

Comment:
On iOS unfortunately the given filename is ignored (but can be changed in the sharing dialog from `file` to anything else). I already opened an issue in the react-native-share project at https://github.com/react-native-share/react-native-share/issues/963 - let me know if I might have missed something on how to configure the filename on iOS.